doc/src/stdlib/collections.md

```diff
@@ -265,3 +266,9 @@ Fully implemented by:
 
   * `Vector` (a.k.a. 1-dimensional [`Array`](@ref))
   * `BitVector` (a.k.a. 1-dimensional [`BitArray`](@ref))
+
+## Utility Collections
```

Comment:
I think we should move this section to the top of this file, and include the other collections in there too. Specifically `Set`, `IntSet` and `Dict` which are all included in weird places in this file.

Reply:
I'm not sure: Associative and Set-like more or less refers to the protocol of implementing those methods. There isn't really anything like that for Pair.